test/built-ins/intrinsics/FunctionPrototype/FunctionPrototype_restricted-property-arguments.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2015 Caitlin Potter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*--- 
+description: Intrinsic %FunctionPrototype% has poisoned own property "arguments"
+---*/
+
+var FunctionPrototype = Function.prototype;
+
+assert.sameValue(FunctionPrototype.hasOwnProperty('arguments'), true, 'The result of %FunctionPrototype%.hasOwnProperty("arguments") is true');
+
+var descriptor = Object.getOwnPropertyDescriptor(FunctionPrototype, 'arguments');
+assert.sameValue(descriptor.configurable, 'function', '%FunctionPrototype%.arguments is a configurable property');
```

Comment:
oop --- there were a few of these somehow, I think my text editor was saving things wrong while I was building content_shell --- keystrokes not registering or something. Anyway, fixing those up as they get noticed